config/index.js

```diff
@@ -387,6 +387,39 @@ var conf = convict({
     format: Number,
     env: 'CLIENT_ADDRESS_DEPTH',
     default: 3
+  },
+  signinConfirmation: {
+    enabled: {
+      doc: 'enable signin confirmation',
+      default: true,
```

Comment:
I wonder if it's worth defaulting this to false, so that we can't accidentally deploy with it turned on full-throttle for the initial version.  Thoughts?